spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/reactive/DefaultRequest.java

```diff
@@ -17,8 +17,30 @@
 package org.springframework.cloud.client.loadbalancer.reactive;
 
 /**
+ * A default implementation of {@link Request}.
+ *
  * @author Spencer Gibb
+ * @author Olga Maciaszek-Sharma
  */
-public class DefaultRequest implements Request {
+public class DefaultRequest implements Request<DefaultRequestContext> {
```

Comment:
I think this should be `<T>` still. And then when instantiated user can use `<DefaultRequestContext>`. Maybe?

Reply:
ok